UPGRADE-4.x.md

```diff
@@ -1,13 +1,30 @@
 UPGRADE 4.x
 ===========
 
-UPGRADE FROM 3.x to 3.x
+UPGRADE FROM 4.x to 4.x
 =======================
 
 ### Support for NelmioApiDocBundle > 3.6 is added
 
 Controllers for NelmioApiDocBundle v2 were moved under `Sonata\UserBundle\Controller\Api\Legacy\` namespace and controllers for NelmioApiDocBundle v3 were added as replacement. If you extend them, you must ensure they are using the corresponding inheritance.
 
+### Fix REST API routing paths
+
+In `SonataUserBundle` v4.6.0 some wrong REST API routing paths was added. You should avoid using it to keep API doc cleaner. Change this one to correct paths:
```

Comment:
```suggestion
In `SonataUserBundle` v4.6.0 some extra REST API routing paths were added. You MUST avoid using them to keep API doc cleaner. To do so, make sure to use the right routing file:
```

Reply:
Do you have some concerns about this suggestion?

Reply:
My bad, I think i marge it.

Reply:
Done.